php-frontend/src/main/java/org/sonar/plugins/php/api/visitors/PHPVisitorCheck.java

```diff
@@ -163,6 +163,7 @@ public void visitParameter(ParameterTree tree) {
     scan(tree);
   }
 
+  //fixme (Lena) : rename method to visitUseTraitDeclaration
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)